.github/workflows/ci.yml

```diff
@@ -29,7 +29,7 @@ jobs:
         with:
           version: ${{ matrix.version }}
           arch: ${{ matrix.arch }}
-      - uses: julia-actions/cache@v1
+      - uses: julia-actions/cache@v2
```

Comment:
Uhm, should have dependabot to automate this (e.g. codecov/codecov-action below should also be v5l

Reply:
hm good point, will try to enable that